clappr/clappr-ios
# Change MediaControl center panel

# Goal

To change the position of `MediaControl` plugins using `center` panel and `none` position

# How to test

1. Create a `MediaControl` plugin like this:

```swift
class MyPlugin: MediaControlPlugin {
    override open var pluginName: String {
        return "myplugin"
    }
    
    override open var panel: MediaControlPanel {
        return .center
    }
    
    override open var position: MediaControlPosition {
        return .none
    }
    
    override func render() {
        view.backgroundColor = .white
        view.frame = CGRect(x: 0, y: 0, width: 50, height: 50)
    }
}
```

2. Then, register it before the player is initiated:

```swift
Loader.shared.register(plugins: [MyPlugin.self])
```

3. Run and check if a white square appears in the front of `MediaControl`
